src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.js

```diff
@@ -89,6 +89,7 @@ class ExistingPaymentMethodsController {
         paymentMethod: existingPaymentMethod,
         disableCardNumber: !!existingPaymentMethod && !existingPaymentMethod['from-current-order'],
         mailingAddress: this.mailingAddress,
+        defaultPaymentType: () => this.defaultPaymentType,
```

Comment:
Resolve is always weird... Idk why the others work. If I don't make it a function, it tries to use it as a component or something and I get `Unknown provider: creditCardProvider <- creditCard`.